freedomofpress/securedrop.org
# Bind docker ports to localhost only

How to test: 

* CI should pass
* Checkout master
* make dev
* open browser to `<IP_OF_HOST>:8000`
* observe web page
* checkout this branch
* make dev
* open browser to `<IP_OF_HOST>:8000`
* do not observe web page

Fixes 1/4 of https://github.com/freedomofpress/fpf-www-projects/issues/19, and updates `cryptography` to pass CI.
